UnitTest/TestComponentCSharp_Tests.cs

```diff
@@ -122,6 +122,30 @@ public void TestDynamicInterfaceCastingOnInvalidInterface()
         {
             Assert.ThrowsAny<System.Exception>(() => (IStringableInterop)(IWinRTObject)TestObject);
         }
+
+        [Fact]
+        public void TestBuffer()
+        {
+            var buffer = new Windows.Storage.Streams.Buffer(2);
+            var array = buffer.ToArray(0, 2);
+            Assert.Equal(2, array.Length);
+        }
+
+        [Fact]
+        public async void TestStorageFile()

```

Comment:
I think our pipeline is going to throw a fit on these tests. From what I've read, xunit and `async void` test methods are not friends. I have some example tests that show how to pull out the async part into an async method, then the [Fact] attributed test method can just invoke it. 

Take a look at this async method and the test that follows it: https://github.com/microsoft/CsWinRT/blob/f91e9624d200d9b863757a8bcffa338f2fc8203c/UnitTest/TestComponentCSharp_Tests.cs#L92

Reply:
you cannot make a test case async - look at the other pattern we use for async tests, where the [Fact]-attributed test case invokes an async DoIt function that awaits, synchronously Wait()-ing the result